configuration.go

```diff
@@ -32,6 +32,7 @@ var (
 	maxRetriesConfig          = &configuration{flag: "max-retries", envFlag: "max_retries", defaultValue: strconv.Itoa(awsClient.DefaultRetryerMaxNumRetries)}
 	defaultDatabaseConfig     = &configuration{flag: "default-database", envFlag: "default_database", defaultValue: ""}
 	defaultTableConfig        = &configuration{flag: "default-table", envFlag: "default_table", defaultValue: ""}
+	enableSigV4AuthConfig     = &configuration{flag: "enable-sigv4-auth", envFlag: "enable_sigv4_auth", defaultValue: "false"}
```

Comment:
The default value should be true. Although SigV4 authentication isn't supported with Prometheus, it's still better to have users explicitly turn it off when using for best security practices.

Reply:
Okay, I'll also update the Launch URLs.

Reply:
I have set the default to `true`, for the connector and SAM template. I have also updated all of the `Launch` URLs to `Launch Without SigV4` and added placeholder links for all missing regions. None of these links will work until the templates in our buckets are updated.